src/app/archive/lib/processor.ml

```diff
@@ -94,43 +116,69 @@ module Token = struct
     }
   [@@deriving hlist, fields]
 
+  let id_to_t = Hashtbl.create (module Int)
+
+  let value_to_id = Hashtbl.create (module String)
+
+  let loaded = ref false
+
+  let load_local_copy (module Conn : CONNECTION) =
```

Comment:
Why `load_local_copy`? Name looks weird, I suggest reducing it to simply `load_copy`